resources/merge_release_branch.py

```diff
@@ -0,0 +1,43 @@
+from subprocess import check_call, CalledProcessError
+from os import chdir
+from pathlib import Path
+
+
+def integrate_release(url, version, working_directory):
+    """
```

Comment:
Validate inputs. Fail early fail often.

Reply:
Added; the working_directory validation is done automatically when constructing the `Path` object.